### ISSUE TITLE
Add a LICENSE.md file

### ISSUE THREAD

**Issue by Bernd-L** [thethoughtemporium/Whose-gene-is-it-anyway#4]
## TL;DR
Consider adding a `LICENSE.md` file to the root of this repository.

By using this name GitHub will automatically be able to detect which one it is and will display it as a badge above the root rile listing (next to the contributors count).

## About the mention in the `README.md`

I realize that the `README.md` file of this project already references the CC BY-SA 4.0.

This issue addresses 1) the need for a `LICENSE.md` file, and 2) the suggestion to choose a different license.

Furthermore, if someone creates a derivative work of your project, they may license it under the terms of the GNU GPL 3, but you won't be able to include their improvements in the original project, because it would violate the GPL (at least to my understanding, I'm not a lawyer) as stated below:

>"CC BY-SA 4.0 is one-way compatible with the GNU GPL version 3: this means you may license your modified versions of CC BY-SA 4.0 materials under GNU GPL version 3, but you may not relicense GPL 3 licensed works under CC BY-SA 4.0.
\- [GNU licenses list, section "ccbysa"](https://www.gnu.org/licenses/license-list.en.html#ccbysa)

## Choosing the right license

I would highly recommend to choose something like the GPL 3 or something like it.
If you're ok with corporations using this stuff without disclosing their changes, then go for the MIT license instead.

I'll not go into any more detail, because there already is a very good resource created by GitHub themselves called "Choose a license":

<https://choosealicense.com/>

My personal favourite is the [AGPL 3 license](https://www.gnu.org/licenses/agpl-3.0.en.html) (of which an official markdown version can be found on the GNU website: <https://www.gnu.org/licenses/agpl-3.0.md>).
It's a license taylored to sharing improvements and empowering user freedom (which I think is important, *especially* for a project about gene editing).

---
Thanks for reading my wall of text;
I'd love to hear your opinion on this.

**Comment by Two-Tone** [comment]
> If you're ok with corporations using this stuff without disclosing their changes, then go for the MIT license instead.

It's not just corporations that do that, but individuals too.  When DOOM 1's source code was first released it had a non copyleft license ([the DSL](https://doom.fandom.com/wiki/Doom_Source_License)) and as a result a lot of source ports don't give access to their source code because the authors wanted to be secretive with it.

**Comment by Bernd-L** [comment]
See my comment on [the commit](https://github.com/robellegate/Whose-gene-is-it-anyway/commit/9431fc9b36a756114a0c8f89d5cb0353f2348cf6
) between the this comment and the one made by @​Two-Tone on (re)-licensing and formatting:

Link to comment:
https://github.com/robellegate/Whose-gene-is-it-anyway/commit/9431fc9b36a756114a0c8f89d5cb0353f2348cf6#commitcomment-39274043
